docs/visual-basic/language-reference/modifiers/protected.md

```diff
@@ -12,17 +12,20 @@ helpviewer_keywords:
 ms.assetid: 74ad3d56-309f-49d2-b60c-1d0157d010e8
 ---
 # Protected (Visual Basic)
-Specifies that one or more declared programming elements are accessible only from within their own class or from a derived class.  
+A member access modifier that specifies that one or more declared programming elements are accessible only from within their own class or from a derived class.  
   
 ## Remarks  
  Sometimes a programming element declared in a class contains sensitive data or restricted code, and you want to limit access to the element. However, if the class is inheritable and you expect a hierarchy of derived classes, it might be necessary for these derived classes to access the data or code. In such a case, you want the element to be accessible both from the base class and from all derived classes. To limit access to an element in this manner, you can declare it with `Protected`.  
+
+> [!NOTE]
```

Comment:
This is great. I'd recommend adding a similar note to "Friend" and "Private".

Feedback I got from the VS team when I made similar changes in C# was that F1 help on `private protected` would load either `private` or `protected`. Similar concerns happened with `protected internal` loading either `protected` or `internal`. The IDE picks the single token under the cursor, rather than the compound word. A link to the other possible correct targets would be useful.